### PR TITLE
✨ Add Richaudience to RTC callout vendors

### PR DESCRIPTION
**Pull request by richaudience**
* Add Richaudience to RTC callout-vendors for amp-ad
* Add Richaudience to RTC documentation
* Add Richaudience to RTC publisher-implementation-guide

* Update callout-vendors.js

### CODE REVIEW

File: extensions/amp-a4a/rtc-documentation.md [hunk 1]
```diff
@@ -149,6 +149,7 @@ The `errorReportingUrl` property is optional. The only available macros are ERRO
 -   [PubWise](https://www.pubwise.io)
 -   Purch
 -   Rubicon
+-   RichAudience
```
Comment (powerivq) [inline extensions/amp-a4a/rtc-documentation.md:152]: Nit: please sort alphabetically (before Rubicon).

File: extensions/amp-a4a/rtc-publisher-implementation-guide.md [hunk 2]
```diff
@@ -43,6 +43,7 @@ To use RTC, you must meet the following requirements:
 -   [PubWise](https://pubwise.io)
 -   Purch
 -   Rubicon
+-   RichAudience
```
Comment (powerivq) [inline extensions/amp-a4a/rtc-publisher-implementation-guide.md:46]: Nit: order

**Review by powerivq — CHANGES_REQUESTED** [review]

File: extensions/amp-a4a/rtc-documentation.md (hunk 1 above)
Comment (richaudience) [reply extensions/amp-a4a/rtc-documentation.md:151]: the change has been applied in the next commit. Please check it.
Cheers

File: extensions/amp-a4a/rtc-publisher-implementation-guide.md (hunk 2 above)
Comment (richaudience) [reply extensions/amp-a4a/rtc-publisher-implementation-guide.md:45]: The change has been applied in the next commit. 
Please check it.
Cheers

**Review by powerivq — APPROVED** [review]

**Comment by powerivq** [comment]
Hi, @richaudience you need to rebase against the latest main branch. The build is failing because of that.

**Pull request merged**